app/containers/message/Components/Audio/Loading.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect } from 'react';
+import Animated, { Easing, useAnimatedStyle, useSharedValue, withRepeat, withTiming } from 'react-native-reanimated';
+
+import { CustomIcon } from '../../../CustomIcon';
+import { useTheme } from '../../../../theme';
+
+const Loading = () => {
```

Comment:
I'm not sure about this animation.
It's far too different from the usual circle animation, because there's no "collapsed" point.
It could be just me, but this is what I expect from it (and that's why I didn't want to use the ActivityIndicator from RN) https://m2.material.io/components/progress-indicators#circular-progress-indicators
You can also take a look at the favicon from Chrome loading.